tests/empty-file/expected_results.json

```diff
@@ -1,5 +1,5 @@
 {
   "version": 1,
   "status": "fail",
```

Comment:
I would have expected this status to be `error`, as an empty file is likely a compile error in Idris, right?
The `error` status is used for when _not a single test could be executed_. The `fail` status is for test that executed but didn't pass.

Reply:
An empty file in itself is not necessarily a compilation error (in the empty file). The tests wont compile however, so it should be an error, because no tests will have run. I will try to have run.sh decide between error and fail.

Reply:
Indeed

Reply:
Fixed!